### DISCUSSION TITLE
Windows Default Sound Icon On Taskbar Disabled

### DISCUSSION THREAD

**Discussion by SpookyMcBones** [irzyxa/Volume2#324] [Q&A] [answered]
Hiya,

I'm on Windows 10. I can only assume it's due to Volume2, but my default sound icon on the taskbar is missing, but in Windows settings for what system icons to show, the volume one is disabled, and blanked out (unable to change). Is there a setting in Volume2 to re-enable it?

Thanks a bunch!

**Comment by irzyxa** [comment, marked as answer]
Please try to run Volume2 as administrator, go to System tab set/reset option _Show standard volume.._ and press _Apply_
![image](https://github.com/irzyxa/Volume2/assets/5119360/a3828f92-ad31-468a-a880-dc43acd3231b)

**Comment by SpookyMcBones** [comment]
Hi,

Thank you so much for your help.

I see that option in my program, however, even when running Volume2 as admin, that option is unable to be checked (blanked out).

**Reply by irzyxa** [reply]
In this case we should try to use RegEdit utility:
https://admx.help/?Category=Windows_10_2016&Policy=Microsoft.Policies.TaskBar2::HideSCAVolume
